README.asciidoc

```diff
@@ -1,16 +1,41 @@
-RPMGrill User Guide
-===================
+RPMGrill
+========
 :doctype: book
 :toc:
 :lang: en
 
+Overview
+--------
+
```

Comment:
I like those because it is good indicator to a potential contributor that there are tests and are run continuously